tools/system_libs.py

```diff
@@ -1543,16 +1546,19 @@ def get_cflags(self):
     cflags += ['-DNDEBUG', '-DEMSCRIPTEN_STANDALONE_WASM']
     if self.is_mem_grow:
       cflags += ['-DEMSCRIPTEN_MEMORY_GROWTH']
+    if self.eh_wasm:
+      cflags.append('-D__USING_WASM_EXCEPTIONS__')
```

Comment:
I think adding `NoExceptLibrary` to the class list will add this automatically. Was there a reason not to do that? I guess it then builds 3 modes and we need just 2, really, but the code would be simpler I think.

Reply:
I did that first but we really don't need 3 different modes here .. and it would make the default `libstandalone-noexcept` which is a pointless library.   I think the duplication here is worth it to avoid the extra combination.. especially when this library already has a `-no-growth` variant.

Reply:
Makes sense, agreed.